roll/__init__.py

```diff
@@ -233,6 +231,7 @@ class Roll:
     def __init__(self):
         self.routes = self.Routes()
         self.hooks = {}
+        from .extensions import options
```

Comment:
We are now importing `HTTPError` from extensions, so cyclic import :/
Should we move `HTTPError` to a third file?

Reply:
Or not apply `options` by default?

Reply:
Tempting!
But I'm afraid each and every Roll usage will need to use it, as I think a web server that does not answers to `options` seems ineffective.

Reply:
I think it's only in case of an API, Roll can just serve regular views fine without responding to OPTIONS requests(?)

Reply:
OK, let's try that way :)